src/freenas/usr/local/lib/middlewared_truenas/plugins/failover.py

```diff
@@ -1537,6 +1551,17 @@ async def hook_setup_ha(middleware, *args, **kwargs):
     except Exception:
         ha_configured = False
 
+    # on brand new HA systems, we don't synchronize service states to the
+    # standby controller. Minimally, however, it's nice to synchronize ssh
+    # (if appropriate, of course)
+    filters = [('srv_service', '=', 'ssh')]
+    ssh_enabled = remote_ssh_started = False
```

Comment:
Shouldnt this whole block be in the `if not ha_configured:` ?

I dont want to bother to keep trying to start ssh all the other times, only if its really the first time we are setting up HA